plugins/dialog/styles/dialog.css

```diff
@@ -10,7 +10,7 @@
 	height: 100%;
 	top: 0;
 	left: 0;
-	z-index: 10010;
+	z-index: 999999 !important;
```

Comment:
Please check this comment https://github.com/oat-sa/tao-core-shared-libs-fe/pull/5#discussion_r453439572